tutorials/git-hooks.md

```diff
@@ -0,0 +1,50 @@
+# Czym są git hooki
+
+Przepływ operacji w gitcie jest dość dobrze znany i ustalony. Wykonujemy kolejno operacje `git add`, `git commit` `git pull` `git push`, etc. Co jednak jeśli chcielibyśmy aby w pewnych sytuacjach działo się coś dodatkowego.
```

Comment:
i interpunkcja w całym pliku, ale to już chyba mało istotne rzeczy, jednak wygodniej się czyta

Reply:
Coś nie coś poprawiłem. Proszę uszanuj mój czas w tym kursie i chociaż wskaż mi miejsca, w których dostrzegasz ten problem.

Reply:
jest okej